synapse/models/infotech.py

```diff
@@ -1255,6 +1259,45 @@ def getModelDefs(self):
                         'doc': 'An array of ATT&CK technique IDs addressed by the mitigation.',
                     }),
                 )),
+                ('it:mitre:attack:campaign', {}, (
```

Comment:
This may already be in the works, but do we intend on adding a ou:campaign:mitre:attack:campaign prop to the model to allow users to correlate mitre campaigns to non mitre campaigns?